students/ABartles/Exercise6/test_mailroom.py

```diff
@@ -0,0 +1,67 @@
+from mailroom import *
+import os
+
+'''
+Items to Test:
+    sum_gifts
+    count_gifts
+    sorted donors
+    list_donors
+    *thanks letter
+    *send_thanks_all
+
+Items with user interaction: (can pytest test/simulate user imput?)
+    rec_donation
+    send_thanks_one
+    menu
+
+Not sure how to test w/ pytest:
+    create_report
```

Comment:
to test create_report you need to split it into two functions:
1. `generate_report` function which can return a list of rows or already formatted string
2. `print_report` function which consumes output from generate report and prints the report


you will write a unit test for `generate_report` function.